### ISSUE TITLE
Unused function argument model_name

### ISSUE THREAD

**Issue by woctezuma** [minimaxir/gpt-2-simple#39]
The argument `model_name` is not used in `generate()`. Here is [the line](https://github.com/minimaxir/gpt-2-simple/blob/873db52d79b380a79cdda0012208a105428e9306/gpt_2_simple/gpt_2.py#L313).

**Comment by TheDailySpank** [comment]
Saw that as well. Was thinking about patching it but I'm at work right now and can't. I just did a quick search and replace "117M" -> "345M" and downloaded the 345M model and it appears to be working.

**Comment by woctezuma** [comment]
For `download()`, it works. Here is [the line](https://github.com/minimaxir/gpt-2-simple/blob/873db52d79b380a79cdda0012208a105428e9306/gpt_2_simple/gpt_2.py#L27).

For `generate()` and `generate_to_file()`, the program relies on a **checkpoint** of the **fine-tuned** model, so there is no need to specify `model_name` as a function argument.

**Comment by minimaxir** [comment]
Correct, `model_name` and `run_name` in `generate` functions are unnecessary and can be removed.
